cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/BoardsConfigService.kt

```diff
@@ -1,34 +1,36 @@
 package com.cognifide.cogboard.config.service
 
 import com.cognifide.cogboard.CogboardConstants.Companion.PROP_CONTENT
+import com.cognifide.cogboard.config.ConfigType.BOARDS
 import com.cognifide.cogboard.config.helper.EntityCleanupHelper
 import com.cognifide.cogboard.storage.ContentRepository
 import com.cognifide.cogboard.storage.Storage
-import com.cognifide.cogboard.storage.VolumeStorageFactory.boards
+import com.cognifide.cogboard.storage.VolumeStorageFactory.get
 import io.vertx.core.json.JsonObject
 import com.cognifide.cogboard.CogboardConstants as CC
 
 class BoardsConfigService(
-    private val storage: Storage = boards(),
     private val contentRepository: ContentRepository = ContentRepository(),
-    private val entityCleanupHelper: EntityCleanupHelper? = null
+    private val entityCleanupHelper: EntityCleanupHelper? = null,
+    private val configFile: String = BOARDS.configFile()
```

Comment:
not clear change for me. `VolumeStorage` is for me kind of `Respository`. Maybe it is worth to change the name to `ConfigRepository` which will keep the config file name/path. Service should not care. It should have ConfgiRepository.